modules/representation_management/spec/lib/representation_management/v0/pdf_constructor/form_2122_spec.rb

```diff
@@ -44,6 +44,14 @@
     }
   end
 
+  before do
```

Comment:
What's the reasoning for mocking active record queries in these specs?

Reply:
I'm not sure what other way I can get the correct string for organization name to match the existing fixture PDF.  Is there a better way to do this?

Reply:
You could use a factory for RepresentationManagement::Form2122Data on line 56

Reply:
This was a great suggestion @stevenjcumming .  Thank you!